src/core/Akka.Persistence.TestKit.Tests/LocalSnapshotStoreSpec.cs

```diff
@@ -14,7 +14,8 @@
 
 namespace Akka.Persistence.TestKit.Tests
 {
-    public class LocalSnapshotStoreSpec : SnapshotStoreSpec
+    // FIXME
+    public abstract class LocalSnapshotStoreSpec : SnapshotStoreSpec
```

Comment:
To skip only LocalSnapshot spec, not all specs inherited from `SnapshotStoreSpec`